jobbergate-api/jobbergate_api/apps/services.py

```diff
@@ -231,6 +231,11 @@ async def clone_instance(self, original_instance: CrudModel, **incoming_data) ->
         data.update(incoming_data)
         data["cloned_from_id"] = original_instance.id
 
+        if "created_at" in data:
+            del data["created_at"]
+        if "updated_at" in data:
+            del data["updated_at"]
```

Comment:
You could use `pop()` here to do these operations in one line:

```suggestion
        data.pop("created_at", None)
        data.pop("updated_at", None)
```

Reply:
Solved by [7883b10](https://github.com/omnivector-solutions/jobbergate/pull/643/commits/7883b10a9462d051e4dd414c781c958d4e9350a5)